includes/discovery/os/avaya-vsp.inc.php

```diff
@@ -2,7 +2,7 @@
 /*
  * LibreNMS
  *
- * Copyright (c) 2016 Neil Lathwood <neil@lathwood.co.uk>
```

Comment:
You can't remove my copyright, it's fine to add your copyright with it as well but you'll need to restore this line.